client/my-sites/importer/importing-pane.jsx

```diff
@@ -141,10 +225,11 @@ export default React.createClass( {
 					/>
 				}
 				{ this.isImporting() && (
-					percentComplete
+					percentComplete >= 0
```

Comment:
@rralian good catch - I addressed this by making the check more complete to determine if the progress information is available